src/main/java/org/apache/commons/imaging/formats/png/PngImageParser.java

```diff
@@ -622,7 +577,9 @@ public BufferedImage getBufferedImage(final ByteSource byteSource, Map<String, O
 
             ScanExpediter scanExpediter;
 
-            switch (pngChunkIHDR.interlaceMethod) {
+            // Note that here we assume the only filter method. Hence, the
+            // filtering and interlacing are combined (correct??)
```

Comment:
For this note a `TODO` is probably better, as the IDE/static analyzers may tell other developers about it. And remind them to revisit this every now and then.